R/plotDataSetList.R

```diff
@@ -685,26 +696,32 @@ Plot.RT.Histogram.DataSetList <- function(dsList, ftarget, plot_mode = 'overlay'
                   marker = list(color = rgba_str,
                                 line = list(color = 'rgb(8,48,107)', width = 1.5)))
     } else if (plot_mode == 'subplot') {
+      disp_y <-  mod(i, n_cols) == 1
+      disp_x <- i > (N - n_cols)
+      disp <- c(disp_x, disp_y)
+      
       p[[i]] %<>%
         add_trace(data = plot_data, x = ~x, y = ~y, width = ~width, type = 'bar',
                   name = algId, text = ~text, hoverinfo = 'text',
                   marker = list(color = rgba_str,
-                                line = list(color = 'rgb(8,48,107)', width = 1.5))) %>%
-        layout(
```

Comment:
There is no need to add layout now?

Reply:
Not for the labels, I changed that to use axis labels instead